src/utils/sorted-electron-map.ts

```diff
@@ -20,12 +20,27 @@ export function sortedElectronMap<T>(
         return a.localeCompare(b);
       }
 
-      if (!semver.valid(a)) {
-        return -1;
-      }
+      if (!semver.valid(a)) return -1;
+      if (!semver.valid(b)) return 1;
+
+      // Handle prerelease tag sorting - semver does not handle this.
+      // See https://www.npmjs.com/package/semver#prerelease-tags for more.
+      if (semver.prerelease(a) && semver.prerelease(b)) {
```

Comment:
```suggestion
      if (semver.major(a) === semver.major(b) && semver.prerelease(a) && semver.prerelease(b)) {
```

Reply:
(maybe check minor/patch versions too? i know we don't currently have any prerelease versions that aren't `x.0.0`, but we might in future have `x.1.0-beta.1` or something so it'd be best to compare the full version here i think)

Reply:
This gets checked on line 32 - happy to move it but logically this should already work as expected?